pkg/kdump/kdump-client.js

```diff
@@ -61,6 +62,19 @@ export class KdumpClient {
 
         // watch the config file
         this.configClient = new ConfigFile("/etc/kdump.conf", true);
+        this._watchConfigChanges();
+
+        this.configClient.wait().then(() => {
+            // if no configuration found, try SUSE version
+            if (this.configClient.settings === null) {
+                this.configClient.close();
+                this.configClient = new ConfigFileSUSE("/etc/sysconfig/kdump", true);
+                this._watchConfigChanges();
+            }
```

Comment:
These 5 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17634-20220810-111027-b2d4a356-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)